website/public/js/controllers/groupsCtrl.js

```diff
@@ -16,6 +16,13 @@ habitrpg.controller("GroupsCtrl", ['$scope', '$rootScope', 'Shared', 'Groups', '
     }
 
     $scope.isMemberOfGroup = function(userid, group){
+
+      // If the group is a guild, just check for an intersection with the
+      // current user's guilds, rather than checking the members of the group.
+      if(group.type === 'guild') {
+        return _.detect(Groups.myGuilds(), function(g) { return g._id === group._id });
```

Comment:
Very glad I wrote those tests, because _.detect doesn't return a boolean value like I thought. Had to adjust it to:
`return _.detect(Groups.myGuilds(), function(g) { return g._id === group._id }) ? true : false;`